azkaban-exec-server/src/main/bash/azkaban-executor-start.sh

```diff
@@ -44,6 +44,10 @@ serverpath=`pwd`
 if [ -z $AZKABAN_OPTS ]; then
   AZKABAN_OPTS="-Xmx3G"
 fi
+# Set the log4j configuration file
```

Comment:
Doesn't this code assume that the conf dir will be under the binary dir? The location can be overwritten by using the -conf parameter eg. -conf temp/conf 

What do you think about adding the conf directory to the search path? Or use PropertyConfigurator see http://stackoverflow.com/questions/7390521/change-location-of-log4j-properties  What's the implication for log4j2?